### PR TITLE
Wrapper for payment object API #1

### PR DESCRIPTION
**Pull request by vroudge**

### CODE REVIEW

File: src/GoCardless.js [hunk 1]
```diff
@@ -70,7 +77,101 @@ export default class GoCardless {
...
+     * @param metadata any data up to 3 pairs of key-values
+     * @param internalReference your own internal reference
+     */
+    createPayment(mandateID, amount, currency = "EUR", charge_date = null, description = null, metadata = null, internalReference = null) {
+
+        const body = {
+            'payments': {
+                'amount': amount,
+                'currency': currency,
+                'charge_date': moment(charge_date, 'YYYY-MM-DD'),
```
Comment (getvega) [inline src/GoCardless.js:100]: can we remove that dependency on momentjs?
wanna keep that lib as slim as possible

File: src/GoCardless.js [hunk 2]
```diff
@@ -70,7 +77,101 @@ export default class GoCardless {
...
+     * https://developer.gocardless.com/pro/2015-07-06/#payments-create-a-payment
+     * @param mandateID REQUIRED ID of the mandate against which payment should be collected
+     * @param amount REQUIRED amount in pence, cents or öre
+     * @param charge_date a future date on which the payment should be collected
+     * @param currency defaults to EUR, either EUR, GBP or SEK
+     * @param description human readable description sent to payer
+     * @param metadata any data up to 3 pairs of key-values
+     * @param internalReference your own internal reference
+     */
+    createPayment(mandateID, amount, currency = "EUR", charge_date = null, description = null, metadata = null, internalReference = null) {
```
Comment (getvega) [inline src/GoCardless.js:94]: charge_date › chargeDate
like other fields

File: src/GoCardless.js (hunk 1 above)
Comment (vroudge) [inline src/GoCardless.js:100]: Yeah no problem, will resubmit PR!

File: src/GoCardless.js (hunk 2 above)
Comment (vroudge) [inline src/GoCardless.js:94]: Oops :+1:

**Comment by vroudge** [comment]
Will resubmit soon

**Pull request closed without merging**